app/backend/systemproperties.cpp

```diff
@@ -119,7 +119,13 @@ void SystemProperties::querySdlVideoInfo()
         }
     }
 
+#ifdef HAVE_EGL
+    SDL_Window* testWindow = SDL_CreateWindow("", 0, 0, 1280, 720, SDL_WINDOW_HIDDEN | SDL_WINDOW_OPENGL);
```

Comment:
I think you can get around having to modify all of these `SDL_CreateWindow()` calls with `SDL_WINDOW_OPENGL` by just creating a dummy SDL renderer in EGLRenderer using `SDL_CreateRenderer()` in EGLRenderer. SDL has a magic internal function called `SDL_RecreateWindow()` which can add `SDL_WINDOW_OPENGL` after the fact.

Reply:
Is https://github.com/xdbob/moonlight-qt/commit/c940fedf244cfb992808c455c43125a49e2b39ed what you have in mind ?

Reply:
Yep, that cleans things up a lot. One minor nit: `m_dummyRenderer` should be `m_DummyRenderer`

Reply:
>< Sorry, I am not accustomed to letter case separations